subsys/kafka/src/main/java/org/commonjava/indy/subsys/kafka/boot/KafkaStreamBooter.java

```diff
@@ -97,4 +93,15 @@ public int getBootPriority()
     {
         return 100;
     }
+
+    private Properties setKafkaProps()
+    {
+        final Properties props = new Properties();
+        props.putIfAbsent( StreamsConfig.APPLICATION_ID_CONFIG, config.getGroup() );
+        props.putIfAbsent( StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, config.getBootstrapServers() );
+        props.putIfAbsent( StreamsConfig.BUFFERED_RECORDS_PER_PARTITION_CONFIG, config.getRecordsPerPartition() );
+
+        logger.debug( "Kafka props: {}", props );
```

Comment:
If this is only called once and no sensitive data in it(I don't remember we have), we can make it INFO level to help debugging